test/integration/targets/ec2_vpc_vpn_facts/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- block:
+
+  # ============================================================
+  - name: test success with no parameters
+    ec2_vpc_vpn_facts:
+      region: '{{ ec2_region }}'
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
```

Comment:
These tests need to take security tokens so they can be run with sts credentials.

Reply:
In general, follow the [module guidelines](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/GUIDELINES.md#aws-credentials-for-integration-tests) as they cover the detail @s-hertel mentions and more.